python/scannerpy/database.py

```diff
@@ -1143,28 +1146,30 @@ def table(self, name: str) -> Table:
         table_name = None
         table_id = None
         if isinstance(name, str):
+            table_name = name
             if name in self._table_name:
-                table_name = name
                 table_id = db_meta.tables[self._table_name[name]].id
-            if table_id is None:
-                raise ScannerException(
-                    'Table with name {} not found'.format(name))
         elif isinstance(name, int):
+            table_id = name
             if name in self._table_id:
-                table_id = name
                 table_name = db_meta.tables[self._table_id[name]].name
-            if table_id is None:
-                raise ScannerException(
-                    'Table with id {} not found'.format(name))
```

Comment:
Why did you remove this check?

Reply:
Since `ScannerStream` allows for references to tables that don't exist.